jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/builders/TravelerMessageFromHumanToAsnConverter.java

```diff
@@ -682,7 +665,7 @@ private static ObjectNode transformNodeLL(JsonNode oldNode) {
       transformedLat = OffsetLLBuilder.offsetLL(latOffset);
       transformedLong = OffsetLLBuilder.offsetLL(longOffset);
       if (deltaText.equals("node-LL")) {
-        deltaText = nodeOffsetPointLL(transformedLat, transformedLong);
+        deltaText = determineNodeOffsetPointLLType(transformedLat, transformedLong);

```

Comment:
suggestion(non-blocking): `determine` is synonymous with `get` as a prefix. `get` is standard, short, and easy to understand.
```suggestion
        deltaText = getNodeOffsetPointLLType(transformedLat, transformedLong);

```

Reply:
Renamed!